data_analysis.ipynb

```diff
@@ -4,19 +4,1336 @@
    "cell_type": "markdown",
```

Comment:
<p>praise: good work on this graph. I think this will be really useful for our analysis later on.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepakkrish212/MATH327-Final/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>